src/Files.App.Controls/Files.App.Controls.csproj

```diff
@@ -12,6 +12,32 @@
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     </PropertyGroup>
 
+    <PropertyGroup>
+        <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+        <Title>Files UI Controls</Title>
+        <Authors>files-community</Authors>
+        <Company>Files Community</Company>
+        <Copyright>Copyright (c) Files Community</Copyright>
+        <PackageReadmeFile>README.md</PackageReadmeFile>
+        <RepositoryUrl>https://github.com/files-community/Files</RepositoryUrl>
```

Comment:
I do not think we should link to the repo; otherwise, some requests on this package may come to us.

Reply:
I don't think it will make a difference.

Reply:
Ideally we wouldn't include it for the reason @0x5bfa stated, however, it can be helpful for developers if they want to locate the source code, therefore I think it's best to include the url.